src/summaries/introduction_to_defi.yaml

```diff
@@ -0,0 +1,100 @@
+- title: Introduction to Decentralised Finance
+  shortTitle: DeFi Intro
+  key: introduction-to-decentralised-finance
+  details: |
+    - DeFi Origin
+      * DeFi is an open and global financial system built for the rise of next revolution in the internet space.
+      * It provides you control, visibility and complete access towards your funds.
+      * DeFi provides opportunities to anyone with an internet connection to access financial services.
+      * It is largely owned and maintained by its users.
+
+    - What is DeFi?
+      * DeFi is a collective term for financial products and services that are accessible to anyone with an internet connection.
+      * With DeFi, the markets are always open and there are no centralized authorities who can block payments or deny you access to anything.
+      * Services that were previously slow and at risk of human error are automatic and safer now that they're handled by code that anyone can inspect and scrutinize.
+
+
+- title: Centralised vs Decentralised Finance
+  shortTitle: CeFi vs DeFi
+  key: centralised-vs-decentralised-finance 
+  details: |
+    Centralised Finance consist of many problems. Some of them are as listed below.
```

Comment:
Other disadvantages
- Not innovative as they are build legacy systems
- In-efficient - The money for running the huge financial institutions come from profits of public.
- Closed - You can't write a simple app on top of bank